.gitignore

```diff
@@ -9,6 +9,13 @@
 /install
 /logs
 
+# Dynamic Reconfigure   
+/src/thunderbots/software/util/parameter/autogenerated
+/src/thunderbots/software/util/parameter/dynamic_parameters.*
```

Comment:
Forgot to remove this since the `dynamic_parameters` file moved into the autogenerated folder. Will wait until we make a decision on where it should be and then make a change to this line.

Reply:
Mat replied below, looks like we're good to move the files out of the `autogenerated` folder

Reply:
Moved files out of `autogenerated` folder and fixed all the imports. 

I used https://stackoverflow.com/questions/115983/how-can-i-add-an-empty-directory-to-a-git-repository/932982#932982 for the empty folders that have auto generated code in them.

Reply:
for `dynamic_reconfigure_manager` and the `autogenerated` folder, the contents of those folders are gitignored

Reply:
I thought we got rid of the `autogenerated` folder (though this definitely makes sense for the `dynamic_reconfigure_manager` folder).

Reply:
The auto generated folder contains the auto generated cfgs, those are really gross they should probabaly be tucked away

Reply:
I removed those lines and pushed it up earlier, but some reason git is still showing them, not sure why

Reply:
looks all good to me now!